src/behaviours/proxy/resolve-proxy/main/electron.injectable.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import electron from "electron";
+
+const electronInjectable = getInjectable({
```

Comment:
`electronInjectable` sounds like something that is common for other stuff. Are we going to move this whenever there is a need?

Reply:
Yes it is general, but it will be moved to more shared scope only when something else requires it.